src/app/clusters/level-control/level-control.cpp

```diff
@@ -94,6 +94,12 @@ static bool areStartUpLevelControlServerAttributesNonVolatile(EndpointId endpoin
 static constexpr size_t kLevelControlStateTableSize =
     EMBER_AF_LEVEL_CONTROL_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT;
 
+struct CallbackScheduleState
+{
+    System::Clock::Milliseconds32 prevDuration;  // The duration of the previous scheduled callback.
```

Comment:
Duration of what part of it?  Running the actual callback function?  The time we ended up waiting?  Something else?